### PR TITLE
Added VPS from TransIP API v5.1

### PR DESCRIPTION
**Pull request by petericebear**

### CODE REVIEW

File: lib/Transip/Model/Snapshot.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ * @author TransIP (support@transip.nl)
+ */
+class Snapshot
+{
+	/**
+	 * The snapshot name
+	 *
+	 * @var string
+	 */
+	public $name = '';
```
Comment (verschoof) [inline lib/Transip/Model/Snapshot.php]: please remove all defaults to string `public $name = '';` to `public $name;`

**Comment by petericebear** [comment]
I changed the defaults of the models in the code.

**Comment by verschoof** [comment]
The project is based on spaces (4x) and you used tabs, please convert this to spaces to stay in line with code of the rest of the project.

**Comment by petericebear** [comment]
I changed the formatting. PHPStrom was reinstalled to the defaults. sorry for the inconvenience

**Pull request merged**